### PR TITLE
[bbcontent] #48 Assign $this->content before calling $this->isGranted() ...

### PR DESCRIPTION
**Pull request by crouillon**
Assign $this->content before calling $this->isGranted() + Optimize granting check

### CODE REVIEW

File: Resources/helpers/bbcontent.php [hunk 1]
```diff
@@ -100,11 +100,7 @@ private function isGranted()
         try {
             $result = (
                 null !== $this->getRenderer()->getApplication()->getBBUserToken()
-                && (
-                    true === $security_context->isGranted('sudo')
-                    || null === $security_context->getACLProvider()
-                    || true === $security_context->isGranted('VIEW', $this->content)
-                )
+                && $security_context->isGranted('VIEW', $this->content)
```
Comment (mickaelandrieu) [inline Resources/helpers/bbcontent.php]: would you mind to call your variable ``$securityContext`` in order to respect some Symfony best practices ?

**Comment by eric-chau** [comment]
@crouillon, can you explain to us what your changes bring please?

**Comment by eric-chau** [comment]
ping @crouillon

**Comment by eric-chau** [comment]
Code review done yesterday with @crouillon, thank you. :+1:

**Pull request merged**